GliaWidgetsTests/Sources/ChatViewModel/ChatViewModelTests.swift

```diff
@@ -726,6 +728,31 @@ class ChatViewModelTests: XCTestCase {
         XCTAssertEqual(viewModel.messagesSection.items.count, 0)
         XCTAssertEqual(viewModel.receivedMessageIds, [])
     }
+
+    func test_liveObservationAllowed() throws {
+        let viewModel: ChatViewModel = .mock()
+        let alertConfirmation: CallViewController.LiveObservationConfirmation = .init(
+            conf: .mock(),
+            accepted: {
+                viewModel.interactor.state = .enqueued(.mock)
+            },
+            declined: {}
+        )
+
+        alertConfirmation.accepted()
+        XCTAssertEqual(viewModel.interactor.state, .enqueued(.mock))
```

Comment:
The same as for CallViewModel tests

Reply:
fixed [95da2c2](https://github.com/salemove/ios-sdk-widgets/pull/787/commits/95da2c2d67e8fd0b3fddcd08b23139fe16fe4026)